source/hassio/haos_common_tasks.markdown

```diff
@@ -0,0 +1,132 @@
+
+There is no configuration required for editing files within your /config directory. In order to enable access to further directories, editing the add-on configuration is required. See the add-on documentation for details. 
+
+---
+
+### Installing and using the File Editor add-on
+
+A more basic and light weight alternative to Visual Studio Code, the File Editor add-on provides access through Hass-Configurator, which is a web-based filesystem-browser and text-editor. YAML files are automatically checked for syntax errors while editing.
+
+There is no configuration required for editing files within your /config directory. In order to enable access to further directories, editing the add-on configuration is required. See the add-on documentation for details. 
+
+---
+
+## Home Assistant OS Snapshots
+
+Snapshots are a compressed archive file (.tar) of your Home Assistant configuration, in addition to any add-on data and configuration. These are easily made in Home Assistant OS from the Supervisor Snapshot panel and are also available as services which can be used in automations. They can be created from the UI or the CLI (see below) and are stored in the /backup directory.
+
+A full snapshot includes the following directories:
+
+* config
+* share
+* add-ons (these are manually installed or created add-ons, and not those installed from the store)
+* ssl
+* media
+
+A partial snapshot consists of any number of the above default directories and installed add-ons.
+
+### Making a Snapshot from the UI
+
+1. Go to Supervisor > Snapshots in the UI
+2. Provide a name for the snapshot.
+3. Choose full or partial.
+4. Choose to password protect or not. Password protected snapshots cannot easily be browsed outside of Home Assistant OS
+5. Click "Create" to begin the snapshot.
+
+### Restoring a Snapshot on a new install
+
+Restoring a snapshot on a fresh install of Home Assistant OS requires that you first upload the snapshot to the backup directory on the new instance. From the Supervisor Snapshot panel, these can be uploaded using the following steps:
+
+1. Go to Supervisor > Snapshots in the UI 
+2. Select "Upload Snapshot" from the icon in the upper right of the page.
+3. Click on the folder icon to navigate to your snapshot .tar file
+
+When the upload is completed, the snapshot panel should display your newly uploaded file. If not, use the "Reload" option from the icon in the upper right of the page. From there you can select it and then choose to restore it in full or in part, by manually selecting individual folders or add-ons.
```

Comment:
After upload, you are presented with the snapshot dialog for the snapshot you uploaded, no need to reload or click the correct snapshot.